### ISSUE TITLE
minor bug

### ISSUE THREAD

**Issue by rmcgibbo** [pandegroup/openmm#771]
This is a bug on python3. It should be `str(err)`, and it's not corrected by 2to3.

https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/testInstallation.py#L19

```
In [10]: sys.version_info
Out[10]: sys.version_info(major=3, minor=4, micro=2, releaselevel='final', serial=0)

In [11]: ImportError('test').message
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-11-9f637026d45c> in <module>()
----> 1 ImportError('test').message
```

**Comment by swails** [comment]
Agreed.  You want to issue a PR for it?

**Comment by rmcgibbo** [comment]
Fixed by #775.
